docs/Development/Testing.md

```diff
@@ -0,0 +1,24 @@
+VIC Testing
+========
+
+The VIC Test Suite includes six main test types:
+
+1.  **unit**:  function level tests. These tests mostly focus on testing the routines in `vic_run` and `drivers/shared_all`.
```

Comment:
mostly, I do test a few things in the cesm and python drivers using this framework.